README.md

```diff
@@ -320,7 +320,7 @@ mpirun -hostfile hostfile -n 3 ./main -m ./models/7B/ggml-model-q4_0.gguf -n 128
 
 ### BLAS Build
 
-Building the program with BLAS support may lead to some performance improvements in prompt processing using batch sizes higher than 32 (the default is 512). BLAS doesn't affect the normal generation performance. There are currently three different implementations of it:
+Building the program with BLAS support may lead to some performance improvements in prompt processing using batch sizes higher than 32 (the default is 512). BLAS doesn't affect the normal generation performance. There are currently various different implementations of it:
```

Comment:
I would say "several" instead of "various". This description is somewhat inaccurate since it is specifically talking about CBLAS implementations (and possibly Accelerate), which only affects prompt processing speed. cuBLAS, hipBLAS, and I believe even CLBlast accelerate token generation as well, and are part of this section.

Reply:
Sure I agree. Changed the wording accordingly :)

Reply:
I still think it should be clarified that this description only applies to the below CPU BLAS implementations (not the GPU ones), specifically the line "BLAS doesn't affect the normal generation performance," which is obviously untrue for cuBLAS.

Reply:
Oh okay good point. Will update